Sources/StreamChat/Workers/Background/DatabaseCleanupUpdater.swift

```diff
@@ -72,7 +72,7 @@ class DatabaseCleanupUpdater: Worker {
     }
 }
 
-private extension ChannelDTO {
+extension ChannelDTO {
```

Comment:
out of scope

Reply:
Extension contains `ChannelDTO .resetLocalData` used in channel list updater to trump a query -> we need this change 🙂